src/FunctionApps/DSTP/src/test/kotlin/gov/cdc/prime/phdi/Utilities/HelperFunctionTest.kt

```diff
@@ -0,0 +1,133 @@
+        val content = readHL7MessagesFromByteArray(multipleMessages)
+
+        // assert that the reader is able to read the bytes and return a string
+        assertSame(String::class.java, content::class.java)
+    }
+
+    // TEST PARSING FUNCTIONALITY
+    @Test
+    fun testConvertBatchMessagesToList() {
+        // test a single message
+        val single = convertBatchMessagesToList(singleMessage)
+        assertEquals(1, single.size)
+
+        // test multiple messages
+        val multiple = convertBatchMessagesToList(multipleMessages)
+        assertEquals(10, multiple.size)
+
+        // test invalid messages
+        // convertBatchMessagesToList makes no effort to determine if 
+        // the data it receives is valid HL7, so it will still chunk
+        // messages even if one or more of the messages are invalid
+        val invalid = convertBatchMessagesToList(mixedMessages)
+        assertEquals(3, invalid.size)
+
+        // test batch file with a single message
+        val batch_single = convertBatchMessagesToList(batchFileSingleMessage)
+        assertEquals(1, batch_single.size)
+
+        // test batch file with multiple messages
+        val batch_multiple = convertBatchMessagesToList(batchFileMultipleMessages)
+        assertEquals(5, batch_multiple.size)
+
+        //test batch file with no messages
+        val batch_empty = convertBatchMessagesToList(batchFileNoMessage)
+        assertEquals(0, batch_empty.size)
+
+        // test pure text file
+        // convertBatchMessagesToList makes no effort to determine if 
+        // the data it receives is valid HL7, so it will still chunk
+        // messages even if the text itself is not HL7. This should result
+        // in a single element being returned, regardless of how large
+        // the text is.
+        val text = convertBatchMessagesToList(loremIpsum)
+        assertEquals(1, text.size)
+    }
+
+    @Test
+    fun testParse() {
+        val processedMessages = parse(mixedMessages)
+        val processedText = parse(loremIpsum)
+
+        // test that the method is returning the type we expect
+        assertSame(ProcessedMessages::class.java, processedMessages::class.java)
+        assertSame(ProcessedMessages::class.java, processedText::class.java)
+
+        // test that it returns the correct number of valid messages
+        assertEquals(2, processedMessages.valid_messages.size)
+        assertEquals(0, processedText.valid_messages.size)
+
+        // test that it returns the correct number of invalid messages
+        assertEquals(1, processedMessages.invalid_messages.size)
+        assertEquals(1, processedText.invalid_messages.size)
+    }
+
+    // TEST CONVERTING FUNCTIONALITY
+    @Test
+    fun testConvertMessageToFHIR() {
+        // for this test, we'll actually want to Mock hitting the FHIR server
+        // so I'll need help getting this set up
+        assertEquals(true, true)
+    }
```

Comment:
```suggestion
```

Reply:
I don't see the difference between the actual and suggested code. Am I missing something, or was this an accidental request?

Reply:
the suggestion removes this test since it's not really doing anything